lottery.py

```diff
@@ -43,6 +44,7 @@ def increase_reviewer_score(self, reviewer):
         if reviewer not in self._reviewers:
             return
         self._reviewers[reviewer] += 1
+        database_helper.update_user_rating(reviewer)
```

Comment:
No need to store ratings in variable anymore. Also we need to fetch minimum rating for users set using database helper